web/client/themes/default/less/sidegrid.less

```diff
@@ -46,6 +46,7 @@
         display: flex;
         flex-direction: column;
         flex-grow: 1;
+        flex-basis: 100px;
```

Comment:
Just checked and it seems to work but I don't understand the reason of 100px.
Why 100px?
I tested locally and replacing:

```css
flex-grow: 1;
flex-basis: 100px;
```
with

```css
flex-grow: 1;
flex-basis: 0;
```

and I'm getting the same result.

Reply:
@allyoucanmap no reason, just a random number